### PR TITLE
updated README.md

### PR DESCRIPTION
**Pull request by mosespace**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes GH-72

## Type of change

Please chose options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:

- Firmware version:
- Hardware:
- Toolchain:
- pnpm:

# Checklist:

- [ ] I have read the [Contributing Guidelines](https://github.com/OSCA-Kampala-Chapter/react-ui/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [x] I have run pnpm run test to check if all tests are passing, and/or pnpm run test -- -u to update snapshots if I created and/or updated React Components.
- [x] I have checked that the build works locally and that npm run build work fine.
- [ ] I've covered new added functionality with unit tests if necessary.
- [x] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](https://github.com/airqo-platform/AirQo-frontend/blob/staging/.github/PULL_REQUEST_TEMPLATE.md#what-are-the-relevant-tickets)" section to [link the issue.](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by mosespace** [comment]
@HassanBahati is this okay?

File: README.md [hunk 1]
```diff
@@ -44,4 +44,4 @@ Want to contribute? Get stared with our [Contributing Guide](https://github.com/
 
 ## License
 
-[MIT](https://github.com/OSCA-Kampala-Chapter/react-ui/blob/main/LICENSE)
+[React UI is licenced under MIT licence](https://github.com/OSCA-Kampala-Chapter/react-ui/blob/main/LICENSE)
```
Comment (HassanBahati) [inline README.md:47]: Could we have only `MIT Licence` as a link

Comment (HassanBahati) [inline README.md:47]:
```suggestion
React UI is licenced under [MIT Licence](https://github.com/OSCA-Kampala-Chapter/react-ui/blob/main/LICENSE)
```

**Review by HassanBahati — CHANGES_REQUESTED** [review]

**Comment by mosespace** [comment]
@HassanBahati  I have staged the changes

**Review by HassanBahati — APPROVED** [review]
looks good.
thanks @mosespace

**Pull request merged**